cms/templates/js/staff-lock-editor.underscore

```diff
@@ -22,4 +22,19 @@
         </p>
```

Comment:
Note to reviewer: we implemented the new hide from TOC checkbox since in the initial proposal it's alongside the  `Hide from learners` visibility option. But now, `staff-lock-editor.underscore` and the rest of staff lock references don't make much sense. It'd be better to rename the staff lock references to be more generic, referencing more generalized visibility settings.

Reply:
Adding that change here would complicate things even further, so I'll open a new PR with the name suggestion. I'll link it here once it's ready.

Reply:
Naming is always going to be hard and often there will be a bit of mismatch. I think it falls out of the scope of the feature to do a refactor so large. Unless you are very strong against it I would suggest to to live the staff-lock classes as they are.